DTS-STN/Service-Canada-Labs
# Cleaning up variables and Dockerfile.

# General Cleanup (Tech debt related)

General cleanup of the following areas:
- Deprecated pages and APIs
- Environment variables (Either duplicates or unused)
- Dockerfile repeated instructions

## Test Instructions

1. Reinstall dependencies (`yarn`)
2. Run a development build `yarn run dev` and navigate to each page of the app, verifying there are no crashes.
3. Try the new docker command in packagelock.json `yarn run docker:build`. Once the build has completed, run the container with `yarn run docker:run` and verify you can navigate to the app via browser. 

## Definition of Done
